draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1023,15 +1057,28 @@ Steps:
 
 ## Interface
 
-The generic interface for deterministic encoding functions to elliptic curves is as follows:
+The generic interface shared by all encodings in this section is as follows:
```

Comment:
This section can be blended with the next section (Notation)

Reply:
That's possible, but I don't see the advantage. Notation is "what do these symbols in the code mean?", Interface is "how do I use this thing?".